daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -918,7 +918,7 @@ private[lf] object SBuiltin {
       mbKey.foreach { case Node.KeyWithMaintainers(key, maintainers) =>
         if (maintainers.isEmpty)
           throw SErrorDamlException(
-            IE.CreateEmptyContractKeyMaintainers(templateId, createArg.toValue, key)
+            IE.CreateEmptyContractKeyMaintainers(templateId, createArgValue, key)
```

Comment:
Just a minor not directly related to the rest of the PR. 
Avoid recomputation already done at line 913